core/app/[locale]/(default)/product/[slug]/_actions/submit-review.ts

```diff
@@ -64,6 +66,8 @@ export async function submitReview(
       };
     }
 
+    revalidateTag(TAGS.reviews);
```

Comment:
I'm trying to poke holes in this approach... since we don't have reCAPTCHA on the form... does this approach leave the merchant open to bots spamming their reviews and reviews data effectively never being cached?